cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/flow/FlowletProgramRunner.java

```diff
@@ -415,9 +415,7 @@ private Integer getBatchSize(Method method, BasicFlowletContext flowletContext)
       String key = batch.key();
       if (!key.isEmpty()) {
         // Try to lookup the value from runtime arguments
-        Map<String, String> args = RuntimeArguments.extractScope(FlowUtils.FLOWLET_SCOPE, flowletContext.getName(),
-                                                                 flowletContext.getRuntimeArguments());
-        String value = args.get(key);
+        String value = flowletContext.getRuntimeArguments().get(key);
```

Comment:
I made the change at https://github.com/caskdata/cdap/pull/6705/files#diff-f81fe1155ac7343104662ea471b32f53R391